client/src/components/SpinnerWrapper/SpinnerWrapper.stories.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+import { SpinnerWrapper } from "./SpinnerWrapper.js";
+
+export default {
+  title: "SpinnerWrapper",
+  component: SpinnerWrapper,
+};
+
+const Template = (args) => <SpinnerWrapper {...args} />;
+
+export const Basic = Template.bind({});
```

Comment:
You should make a story for each spinner config found in `LeafSpinner.js`, `leaf_spinner_configs`!